src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -68,11 +68,17 @@ public function getConfigTreeBuilder()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
                 ->booleanNode('always_authenticate_before_granting')->defaultFalse()->end()
                 ->booleanNode('erase_credentials')->defaultTrue()->end()
+                ->arrayNode('authorization_checker')
+                    ->canBeEnabled()
```

Comment:
I'm not sure how to make this a little bit less verbose. Any suggestion on the tree structure?